admin_guide/node-allocatable-resources.adoc

```diff
@@ -0,0 +1,58 @@
+= Node Allocatable resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+To provide more reliable scheduling and minimize node resource overcommitment,
+each node has a chance to specify how much resources get consumed
```

Comment:
each node has a change to reserve a portion of its resources for use by kubernetes related components